examples/Demo/FluentUI.Demo.Client/Documentation/Components/List/Select/FluentSelect.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Select
+route: /List/Select
+---
+
+# Select
+
+The `Select` component allows one option to be selected from multiple options.
```

Comment:
A `Select` lets people choose a single option from a list of at least four options. Selects use browser native styling for the listed options. They are ideal for data submission in forms and ease of use on mobile platforms.